### DISCUSSION TITLE
CTRL+SHIFT+RIGHT/LEFT opens only one extension files

### DISCUSSION THREAD

**Discussion by yatoya** [sumatrapdfreader/sumatrapdf#2394] [Q&A]
I have mixed files in my folder. Some have an EPS and some AI extension.
The problem is Sumatra opens only EPS files using CTRL+SHIFT+RIGHT/LEFT shortcut.
It completely skips AI files even though it can display them perfectly.

**Comment by GitHubRulesOK** [comment]
EPS is supported by an installed Ghostscript
AI depending on type of AI is not.
Thus AI is not a supported extension. Really those AI that are PDF could (should ?) be renamed as hybrid.ai.pdf  (personally think many PDF should be renamed xclusive.pdf.xfa but that's a different rant.)
